pandas/io/formats/style.py

```diff
@@ -73,6 +73,12 @@ class Styler:
         List of {selector: (attr, value)} dicts; see Notes.
     uuid : str, default None
         A unique identifier to avoid CSS collisions; generated automatically.
+    uuid_len : int, default 5
```

Comment:
Minor but should append this to the end of the signature, in case people are calling positionally